documentation/src/main/markdown/releasenotes.md

```diff
@@ -16,7 +26,7 @@
 
 ### New features
 
-* Ephemeral Scan, or Ephemeral Scan Mode, is a new way of running [solution_name] with [blackduck_product_name]. This mode is designed to be as fast as possible and does not persist any data on [blackduck_product_name]. See the [Ephemeral Scans page](downloadingandrunning/ephemeralscan.md) for further details.
+* Ephemeral Scan, or Ephemeral Scan Mode, is a new way of running [solution_name] with [blackduck_product_name]. This mode is designed to be as fast as possible and does not persist any data on [blackduck_product_name]. See the [Ephemeral Scans page](downloadingandrunning/statelessscan.md) for further details.
 * The output for Rapid and the new Ephemeral Scan Modes will now include upgrade guidance for security errors and warnings.
```

Comment:
Why two release notes?
If this is just a name change, I would think it would belong only in the Changed features list.

Reply:
As far as I could tell the text of prior release notes does not change, however the page was renamed which broke the documentation build which necessitated fixing up the page url for the old ephemeral scan note.... should I have NOT renamed the page?

Reply:
Oh, I see... sorry, I hadn't noticed that one of these release notes was from the earlier version.